src/app/content/highlights/components/Confirmation.tsx

```diff
@@ -45,15 +45,16 @@ interface Props {
   onConfirm?: () => void;
   onCancel: () => void;
   always?: () => void;
+  drawFocus?: boolean;
 }
 
 // tslint:disable-next-line:variable-name
 const Confirmation = React.forwardRef<HTMLElement, Props>((
-  {message, confirmMessage, confirmLink, always, onCancel, onConfirm, ...props}: Props,
+  {message, confirmMessage, confirmLink, always, onCancel, onConfirm, drawFocus = true, ...props}: Props,
   ref
 ) => {
   return <Overlay
-    ref={mergeRefs(ref, useDrawFocus())}
+    ref={mergeRefs(drawFocus ? ref : null, useDrawFocus())}
```

Comment:
This feels weird... Shouldn't it be always using the passed in ref and sometimes using the drawfocus ref? This is doing it backwards?

Also is drawfocus a hook? It might complain about being conditional, you could extract it to a variable and conditionally compose the variable (I think that'd work anyway)

Reply:
Hmm yeah maybe I flipped this around and didn't notice in testing, I've simplified it as you suggested